client/i18n/locales/english/intro.json

```
@@ -2832,7 +2832,10 @@
           "In these lecture videos, you will learn about modules, imports, and exports in JavaScript."
         ]
       },
-      "lnmg": { "title": "179", "intro": [] },
+      "lab-build-a-password-generator": {
```

```suggestion
      "lab-password-generator": {
```

dashed names for lab bloks are like this